vocabularies/qld-mining-permits.ttl

```diff
@@ -0,0 +1,305 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix minpermit: <http://linked.data.gov.au/def/qld-mining-permit/> .
```

Comment:
prefix at 9 chars? just _permit:_?

Reply:
Yes, prefix should be fewer characters. However, it is more a style thing than a broken thing.
@KellyVance I recommend that we leave as is for now.

Reply:
Yes, should be less characters.
@KellyVance recommend that we keep as is - it is more style thing than a broken thing.